tests/acceptance/test_primer.py

```diff
@@ -0,0 +1,53 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+import contextlib
+import os
+import subprocess
+import sys
+
+import pytest
+from git import Repo
+
+PRIMER_DIRECTORY = ".pylint_primer_tests"
+
+
+@contextlib.contextmanager
+def _patch_stdout(out):
+    sys.stdout = out
+    try:
+        yield
+    finally:
+        sys.stdout = sys.__stdout__
+
+
+def lazy_git_clone(git_url: str, target_directory: str) -> None:
+    if not os.path.exists(target_directory):
+        Repo.clone_from(git_url, target_directory)
+
+
+@pytest.mark.acceptance
+@pytest.mark.parametrize(
+    "git_url,directories_to_lint",
+    [
+        ["https://github.com/psf/black.git", "src tests"],
+        ["https://github.com/numpy/numpy.git", "numpy"],
+    ],
+)
+def test_primer(git_url, directories_to_lint):
+    parts = git_url.split("/")  # ["https:", "github.com", "psf" "black.git"]
+    namespace = parts[-2]  # "psf"
+    project = parts[-1][:-4]  # "black"
+    target_directory = f"{PRIMER_DIRECTORY}/{namespace}/{project}"
+    lazy_git_clone(git_url, target_directory)
+    os.chdir(target_directory)
+    try:
+        # We only check for crash and errors as warning are not stable
+        # We suppose that errors in a big lib come from a pylint false positive
+        subprocess.run(
+            ["pylint"] + directories_to_lint.split(" ") + ["--errors-only"],
+            check=True,
+        )
+    except SystemExit as ex:
+        assert ex.code == 0
```

Comment:
What if a `pylintrc` of a repo specifies that it should fail below < 10 and a new warning lowers the score? `test_lib_module_no_crash` seems to test for `ex.code == 32`. Don't we want to do that here as well?

Reply:
We're using the `--error-only` option, (ie. No warning Convention or refactor messages). If pylint detects an error in a  big well tested lib then it's safe to assume it's a false positive from pylint. We might also launch once and check with the binary exit.

Reply:
Ah I missed that!